packages/web/src/Components/Select/Select.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import MaterialSelect from '@material-ui/core/Select';
+import { Icon } from '../Icon';
+import { MenuItem } from '../MenuItem';
+import InputStateHelpTextProvider from '../InputStateHelpTextProvider';
+import { ISelectProps } from './Select.props';
+import { MenuProps } from '@material-ui/core';
+
+export { ISelectProps } from './Select.props';
+
+/**
+ * ## Importing
+ *
+ * ```
+ * import { Select } from '@naturacosmeticos/natds-web';
+ * ```
+ */
+
+export const Select = React.forwardRef<HTMLSelectElement | HTMLInputElement, ISelectProps>(
+  (props: ISelectProps, ref) => {
+
+    const { state, id, placeholder, options, ...otherProps } = props;
+
+    const [value, setValue] = React.useState('');
+    
+    const handleChange = (event: React.ChangeEvent<{ value: unknown }>) => {
+      setValue(event.target.value as string);
+    };
+
+    const menuProps: Partial<MenuProps> = {
+      anchorOrigin: {
+        vertical: 'bottom',
+        horizontal: 'left',
+      },
+      transformOrigin: {
+        vertical: 'top',
+        horizontal: 'left',
+      },
+      getContentAnchorEl: null,
+    };
+    return (
+      <InputStateHelpTextProvider {...otherProps} state={state}>
+        <MaterialSelect
+          {...props}
+          id={id}
+          displayEmpty
+          IconComponent={(props) => (<Icon name="outlined-navigation-arrowbottom" {...props} />)}
+          onChange={handleChange}
```

Comment:
fico na duvida se a gente não deveria expor o `onChange` pras usuárias.
fico imaginando casos do tipo, "precisamos validar algo quando a usuária escolher uma opção" ou "precisamos ocultar/mostrar/trocar props de outros componentes quando a seleção for X"
como esta (se eu entendi corretamente) a usuária so tem acesso ao valor?

Reply:
Concordo! Acabei de adicionar